src/python/bot/fuzzers/ml/gradientfuzz/constants.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""libFuzzer Neural Smoothing - Globals."""
+
+__author__ = 'Ryan Cao (ryancao@google.com)'
+
+import tensorflow.keras as keras
+import bot.fuzzers.ml.gradientfuzz.models as models
+from datetime import datetime
```

Comment:
Imports should be separated into 3 categories: standard, third party, project specific (https://google.github.io/styleguide/pyguide.html#313-imports-formatting)

i.e.

```python
from datetime import datetime
import enum

import tensorflow.keras as keras

import bot.fuzzers.ml.gradientfuzz.models as models
```

Same applies to all other files in this PR.

Reply:
I see -- will fix!